docs/DxcOnLinux.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+DirectXShaderCompiler on Linux & Mac OS
+=======================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+============
+
+Due to several `requests <https://github.com/Microsoft/DirectXShaderCompiler/issues/1236>`_
+from the community, we have started the effort to enable compilation and running
+of DirectXShaderCompiler on non-Windows platforms (Linux and Mac OS).
+
+
+Building DirectXShaderCompiler
+==============================
+You can follow these steps to build DirectXShaderCompiler on Linux/MacOS:
+
+(We are assuming you have cloned the repository to `~/DirectXShaderCompiler`)
+
+
+.. code:: sh
+
+  cd ~/DirectXShaderCompiler
+  mkdir build && cd build
+  cmake .. -GNinja                                 \
```

Comment:
Typically we want the commands to be runnable directly so that users and copy and run without modification. So I think we only need `cmake .. -GNinja -DCMAKE_BUILD_TYPE=Release $(cat ...)` here. Then we explain further that what the `cmake-predefined-config-params` file is and how to change compilers. `CMAKE_BUILD_TYPE` is relatively well known, so I think we can skip it.

Reply:
Done